### DISCUSSION TITLE
@tanstack/react-table support

### DISCUSSION THREAD

**Discussion by prasannamestha** [petyosi/react-virtuoso#863] [General]
Hello, I am trying to use react-virtuoso with @tanstack/react-table. Has anybody used this?

The docs seems to be using react-table (which has a different api)

Thanks

**Comment by alexalannunes** [comment]
I got it 👍 
https://codesandbox.io/s/react-table-v8-virtuoso-demo-9f0r57?file=/src/App.tsx

**Reply by prasannamestha** [reply]
Thank you :)
